docs/source/operations/besu_networkyaml.md

```diff
@@ -39,6 +39,11 @@ The sections in the sample configuration file are
 
 `version` defines the version of platform being used. The current Hyperledger Besu version support is for **21.10.6** and **22.10.2**.
 
+`permissioning` section contains the flag to enable permissioning in the network
```

Comment:
Please update the above screenshot as well

Reply:
Still this is unclear from the docs, where this `permissioning` field resides.
If we don't expect any other field inside permissioning block, its better to straightaway make this `permissioning: enabled/disabled` (vs the permissioning.enabled: true/false)